src/config/config.go

```diff
@@ -59,15 +50,30 @@ const (
 
 	ZarfComponentsDir = "components"
 
-	ZarfInClusterContainerRegistryNodePort = 31999
-
-	ZarfInClusterGitServiceURL      = "http://zarf-gitea-http.zarf.svc.cluster.local:3000"
-	ZarfInClusterArtifactServiceURL = ZarfInClusterGitServiceURL + "/api/packages/" + ZarfGitPushUser
-
 	ZarfDeployStage = "Deploy"
 	ZarfCreateStage = "Create"
 )
 
+// Zarf Constants for In-Cluster Services.
+const (
+	IPV4Localhost string = "127.0.0.1"
+
+	ZarfArtifactTokenName string = "zarf-artifact-registry-token"
+
+	ZarfImagePullSecretName string = "private-registry"
+	ZarfGitServerSecretName string = "private-git-server"
+
+	ZarfRegistryPushUser                   string = "zarf-push"
+	ZarfRegistryPullUser                   string = "zarf-pull"
+	ZarfInClusterContainerRegistryNodePort int    = 31999
+
+	ZarfGitPushUser string = "zarf-git-user"
+	ZarfGitReadUser string = "zarf-git-read-user"
+
+	ZarfInClusterGitServiceURL      string = "http://zarf-gitea-http.zarf.svc.cluster.local:3000"
+	ZarfInClusterArtifactServiceURL string = ZarfInClusterGitServiceURL + "/api/packages/" + ZarfGitPushUser
```

Comment:
```suggestion
	IPV4Localhost  = "127.0.0.1"

	ZarfArtifactTokenName  = "zarf-artifact-registry-token"

	ZarfImagePullSecretName  = "private-registry"
	ZarfGitServerSecretName  = "private-git-server"

	ZarfRegistryPushUser                    = "zarf-push"
	ZarfRegistryPullUser                    = "zarf-pull"
	ZarfInClusterContainerRegistryNodePort     = 31999

	ZarfGitPushUser  = "zarf-git-user"
	ZarfGitReadUser  = "zarf-git-read-user"

	ZarfInClusterGitServiceURL       = "http://zarf-gitea-http.zarf.svc.cluster.local:3000"
	ZarfInClusterArtifactServiceURL  = ZarfInClusterGitServiceURL + "/api/packages/" + ZarfGitPushUser
```

The types are a bit redundant

Reply:
the types of these things are not shown in VSCode unless you put them there

Reply:
It shows them for me...
![image](https://github.com/defenseunicorns/zarf/assets/3977569/0706fdca-a090-4598-8c2f-72afc35327e3)


If we want to have them we should have them everywhere to stay consistent.

Reply:
im fine w/ that